src/Parser/Mage/Frost/Modules/Features/Abilities.js

```diff
@@ -170,23 +167,9 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.FROST_NOVA,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
-        isOnGCD: true,
-        enabled: !combatant.hasTalent(SPELLS.ICE_WARD_TALENT.id),
         cooldown: 30,
+        enabled: !combatant.hasTalent(SPELLS.ICE_WARD_TALENT.id),
```

Comment:
Did you mean to change this to `charges: combatant.hasTalent(SPELLS.ICE_WARD_TALENT.id) ? 2 : 1,` like you did for fire?

Reply:
I did. I wasnt paying attention and I copied frost nova from fire after i had done it and forgot that i switched branches so the changes i made to frost nova in fire wasnt there.

Its fixed now